include/LightGBM/config.h

```diff
@@ -967,7 +967,8 @@ struct Config {
   // desc = **Note**: refer to `GPU Targets <./GPU-Targets.rst#query-opencl-devices-in-your-system>`__ for more details
   int gpu_device_id = -1;
 
-  // desc = set this to ``true`` to use double precision math on GPU (by default single precision is used in OpenCL implementation and double precision is used in CUDA implementation)
+  // desc = set this to ``true`` to use double precision math on GPU (by default single precision is used)
+  // desc = **Note**: can be used only in OpenCL implementation, in CUDA implementation only double precision is supported yet
```

Comment:
```suggestion
  // desc = **Note**: can be used only in OpenCL implementation, in CUDA implementation only double precision is currently supported
```

Reply:
Fixed in 4a175a9625fe2542abb600bcd6647927157b4dbe.